py/examples/image_annotator_smart.py

```diff
@@ -0,0 +1,80 @@
+# Form / Image Annotator / Events
+# Register the `click` and `tool_change` #events to emit Wave event when the image is clicked or a tool is changed.
+# #form #annotator #image #events
+# ---
+from h2o_wave import main, app, Q, ui
```

Comment:
Let's put this example into `events_click`. Put a text on top saying - "Try annotating the car and see what happens!".